components/server/src/bitbucket-server/bitbucket-server-context-parser.spec.ts

```diff
@@ -186,6 +186,33 @@ class TestBitbucketServerContextParser {
         });
     }
 
+    @test async test_commit_context_02() {
```

Comment:
nit: not necessary to change, just find it hard to navigate through the different test cases. here we're talking about the Commits page of BBS, not the page to show the single commit.

Reply:
No need to change nits right away, it's interrupting testing with the preview env :-(

Reply:
Agree the test name is confusing, changed it to "branch_context"